drm/rp1: rp1-dsi: Add support for inverting lane polarities

6by9 opened the pull request:
The D-PHY on RP1 support lane polarity swapping, and there is a standard device tree mechanism for configuring this, so tie the two together.

https://forums.raspberrypi.com/viewtopic.php?t=388879

6by9 committed:
drm/rp1: rp1-dsi: Add support for inverting lane polarities

The D-PHY on RP1 support lane polarity swapping, and there
is a standard device tree mechanism for configuring this,
so tie the two together.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>

pelwell commented on drivers/gpu/drm/rp1/rp1-dsi/rp1_dsi_dsi.c:
```diff
@@ -145,11 +145,18 @@
 #define DSI_PHYRSTZ_FORCEPLL_LSB 3
 #define DSI_PHYRSTZ_FORCEPLL_BITS  BIT(DSI_PHYRSTZ_FORCEPLL_LSB)
 
+#define DPHY_CLK_PN_SWAP		0x35
```
I could understand putting this register out in front if the other registers were in numerical order, but they aren't.

6by9 replied:
Registers now in order, and lots of whitespace issues and other cruft sorted.

6by9 committed:
drm/rp1: rp1-dsi: Put all register defines into order

Put particularly the PHY registers into order, bitmasks
defined alongside the registers, and Use tabs for indentation.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>

6by9 committed:
drm/rp1: rp1-dsi: Remove all the unused boilerplate register defines

There was lots of register definition information dumped from
the some source into the driver but unused. Remove it, and
format the remaining lines according to the Linux kernel coding
style.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
